TabloidCLI/UserInterfaceManagers/TagManager.cs

```diff
@@ -46,22 +51,81 @@ public IUserInterfaceManager Execute()
 
         private void List()
         {
-            throw new NotImplementedException();
+            List<Tag> tags = _tagRepository.GetAll();
+            foreach (Tag tag in tags)
+            {
+                Console.WriteLine(tag);
+            }
         }
 
+        private Tag Choose(string prompt = null)
```

Comment:
why are we setting this to null in the parameters? I believe even if you did have a string to pass it would be overrridden.